frontend/pubspec.yaml

```diff
@@ -23,6 +23,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  intl: ^0.16.1
```

Comment:
I did not spot the usage of this. Was this added by purpose?

Reply:
Yep, it's used for NumberFormat (formatting the distance)